pkgs/essential-minimal.nix

```diff
@@ -1,12 +1,14 @@
 # The minimal essential tools for app dev.
 { essential-builder
 , essential-node
+, essential-rest-client
 , pint
 , symlinkJoin
 }:
 symlinkJoin {
   name = "essential-minimal";
   paths = [
+    essential-rest-client
     essential-builder
     essential-node
     pint
```

Comment:
This is probably fine, but we should re-think the names of these `devShell`s and think about exactly what we want to expose at some point as "minimal" is probably a bit ambiguous and its not clear where to draw the line